src/components/Cards/ForgottenPassword.vue

```diff
@@ -60,7 +71,7 @@ export default {
 
       this.$store
         .dispatch('forgottenPassword', { email })
-        .then(() => this.$router.push('/'))
+        .then(() => { this.success = true })
```

Comment:
I think this actually makes it a little bit less user friendly as every request now is considered a success, even though the input data isn't accepted by the api.

Reply:
![image](https://user-images.githubusercontent.com/352149/187394059-36bf7a1d-5d9c-49f1-a381-7625ab2aeef8.png)

Reply:
If you write `() => this.succes = true`, it will cause some problem with linting but resolve the problem, so I have to suppress linting in that line.

Reply:
I stared at this a bit more and fundamentally this logic just just isn't really sound at all. I'm pretty sure we need to be replying on the state *way more* for this all to work exactly as intended